docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       - 25556:5556
     environment:
       BEACON_NODE_ENDPOINT: http://localhost:5052
+      GOOGLE_STORAGE_API_ENDPOINT: http://localhost:4443
```

Comment:
We should pass the new env vars too. Maybe not all of them for the development docker-compose but just these:

```
GOOGLE_STORAGE_ENABLED=true
PRISMA_STORAGE_ENABLED=true
```

Reply:
Oops, forgot to do this